portfolio/src/main/java/com/google/sps/servlets/ProximityPin.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.util.StringJoiner;
+
+public final class ProximityPin {
```

Comment:
What is the difference (functionally) between this class and UserComment?
Why have two classes that both represent pins to the user?
Should we have extended UserComment?
Should ProximityPin be a subclass of UserComment?

unrelated to this PR: why is UserComment still called UserComment?

Reply:
The class is still called UserComment because it also stores other information than just the submitted location (i.e. name, email, description, and timestamp) that encompasses a comment. I access the location variable in other places and when I will be implementing pin tags I believe that keeping the name UserComment isn't a misnomer for what the class represents.

These two classes are very close functionality wise but still collect different data from separate sources. UserComment collects more information about the _user_ like their name, while ProximityPin collects information from the _entities_ stored in the datastore. 

I've thought about extending UserComment to ProximityPin but I don't believe it would help the functionality of either class very much, they collect different types of data and don't have any methods within either class that the other could inherit. I use these classes so I can access certain instance variables later on when I may be adding description tags or creating another filtering feature.